### PR TITLE
Users/rok/parallel window machine file copy bug

### PR DESCRIPTION
**Pull request by rokmicrosoft**

### CODE REVIEW

File: Tasks/WindowsMachineFileCopy/RoboCopyJob.ps1 [hunk 1]
```diff
@@ -5,9 +5,13 @@ param (
     [string]$targetPath,
     [object]$credential,
     [string]$cleanTargetBeforeCopy,
-    [string]$additionalArguments
+    [string]$additionalArguments,
+    [string]$scriptRoot
     )
 
+    Import-Module "$scriptRoot\ps_modules\VstsTaskSdk" 
+    Import-VstsLocStrings -LiteralPath $scriptRoot/Task.json
```
Comment (Ajay-MS) [inline Tasks/WindowsMachineFileCopy/RoboCopyJob.ps1]: type of slash differs in two import statement. Can you verify if the localized string is correctly interpreted?

File: Tasks/PowerShellOnTargetMachines/PowerShellJob.ps1 [hunk 2]
```diff
@@ -9,9 +9,12 @@ param (
     [string]$httpProtocolOption,
     [string]$skipCACheckOption,
     [string]$enableDetailedLogging,
-    [string]$sessionVariables
+    [string]$sessionVariables,
+    [string]$scriptRoot
```
Comment (Ajay-MS) [inline Tasks/PowerShellOnTargetMachines/PowerShellJob.ps1]: Increase path version in both tasks.

File: Tasks/WindowsMachineFileCopy/RoboCopyJob.ps1 (hunk 1 above)
Comment (rokmicrosoft) [reply Tasks/WindowsMachineFileCopy/RoboCopyJob.ps1]: I have checked by adding wrong source path . Message is : Source path 'C:\agent\_work\r1\a\$' does not exist. It is correctly interpreted. I have copied from other file it has also used same slash for each import .

File: Tasks/PowerShellOnTargetMachines/PowerShellJob.ps1 (hunk 2 above)
Comment (rokmicrosoft) [reply Tasks/PowerShellOnTargetMachines/PowerShellJob.ps1]: Done . Increased in task Json file patch version.

**Pull request merged**